### PR TITLE
Library needed by SDRangel

### PR DESCRIPTION
**Pull request by ra1nb0w**
#### Description

This PR is a collection of ports needed by SDRangel (and maybe others HAM software). The latter will be sent in the future (I am rewriting all cmake files).

- cm256cc: Fast GF(256) Cauchy MDS Block Erasure Codec in C++
- mbelib: P25 Phase 1 and ProVoice vocoder
- dsdcc: Digital Speech Decoder (DSD) rewritten as a C++ library
- codec2: Codec 2 is an open source speech codec designed for communications quality speech between 700 and 3200 bit/s.
- freedv-gui: (not strictly needed by SDRangel) GUI Application for FreeDV – an open source digital voice protocol that integrates the modems, codecs, and FEC.
- perseus-sdr: Perseus Software Defined Radio Control Library

Template used in accordance with michaelld.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 10.14.5 18F132
Xcode 10.2.1 10E1001

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: audio/mbelib/Portfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+github.setup        szechyjs mbelib 1.3.0 debian/
+checksums           rmd160  d14359c99e8191d25654f792b36590e7372a3990 \
+                    sha256  b7f2e48722028939841606e5292e55844e2112e3a27a2646e674d7ff6055d15d \
+                    size    400249
+revision            0
+
+notes {
+    This source code is provided for educational purposes only.
+    It is a written description of how certain voice encoding/decoding
+    algorythims could be implemented.  Executable objects compiled or
```
Comment (mf2k) [inline audio/mbelib/Portfile]: "algorythims" is misspelled.

File: science/freedv-gui/Portfile [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+github.setup        drowe67 freedv-gui 9b66784d31102d7d63cc5f67208e24fff95beb66
+version             1.3.1
+checksums           rmd160  222b7b15f4fe8b7c427506a48a20e82fe1c17be4 \
+                    sha256  e8da1d5843cebd9b4c9e53a604eec7c26ce00d2f70c5ebb049bfb893199003af \
+                    size    618399
+revision            0
+
+depends_build-append \
+    port:pkgconfig \
+    port:subversion
```
Comment (mf2k) [inline science/freedv-gui/Portfile]: Why is subversion a dependency when the GitHub portgroup is being used?

File: audio/mbelib/Portfile (hunk 1 above)
Comment (ra1nb0w) [reply audio/mbelib/Portfile]: thank you!

File: science/freedv-gui/Portfile (hunk 2 above)
Comment (ra1nb0w) [reply science/freedv-gui/Portfile]: ops; was a reminiscence of using subversion. thank you!

**Comment by cjones051073** [comment]
Can you please check the failed build logs above. Freedv-gui seems to be systematically failing due to a missing dependancy.

**Comment by ra1nb0w** [comment]
> Can you please check the failed build logs above. Freedv-gui seems to be systematically failing due to a missing dependancy.

My fault. Now build fine.

**Comment by cjones051073** [comment]
Thanks, all seems fine now.

**Comment by ra1nb0w** [comment]
Thank you!

**Pull request merged**